websites/0-9/0gomovies/iframe.ts

```diff
@@ -0,0 +1,19 @@
+const iframe = new iFrame();
+
+let sendback: {
+	currTime: number;
+	duration: number;
+	paused: boolean;
+};
+
+iframe.on("UpdateData", () => {
+	if (document.querySelector("video")) {
+		const video: HTMLVideoElement = document.querySelector("video");
+		sendback = {
+			currTime: video.currentTime,
+			duration: video.duration,
+			paused: video.paused,
+		};
+		iframe.send(sendback);
+	}
+});
```

Comment:
To be honest, I'm not sure the variable `sendback` is really *that* usefull since it's used only once.